### ISSUE TITLE
Time picker does not pick the time when selecting using keyboard navigation {{enter}}

### ISSUE THREAD

**Issue by MaartenCradeq** [livewire/flux#2252]
### Flux version

v2.10.0 (not present on at least v2.9.1)

### Livewire version

--

### Tailwind version

--

### Browser and Operating System

Chromium on macOS

### What is the problem?

When using keyboard navigation to select a time by pressing {{enter}} in the Time picker select, the time does not get selected. The console error `Cannot read properties of undefined (reading 'isDisabled')` is thrown.

### Code snippets to replicate the problem

```blade
<flux:time-picker />
```

### Screenshots/ screen recordings of the problem

https://github.com/user-attachments/assets/3d5af249-a888-4b28-9a21-e5df5ef2e369

(The other error `flux.min.js?id=2673235e:125 Uncaught TypeError: Cannot read properties of null (reading 'scrollIntoView')` is explained here #2249 )

### How do you expect it to work?

That pressing enter selects the time.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by ganyicz** [comment]
Hey @MaartenCradeq,

thanks a lot with reporting all these issues, very much appreciated!

Submitted a PR for this as well.

**Comment by MaartenCradeq** [comment]
@ganyicz thanks for all the immediate actions and fixes!
